index.bs

```diff
@@ -7655,7 +7587,7 @@ values are represented by ECMAScript Object values.
     An ECMAScript value |V| is [=converted to an IDL value|converted=]
     to an IDL {{object}} value by running the following algorithm:
 
-    1.  If <a abstract-op>Type</a>(|V|) is not Object, then [=ECMAScript/throw=] a {{ECMAScript/TypeError}}.
+    1.  If <a abstract-op>Type</a>(|V|) is not Object, then [=ECMAScript/throw=] a {{TypeError}}.
```

Comment:
This is another name clash between ES and Web IDL. Previously it linked to ECMAScript, but now it goes to Web IDL. It's mostly fine since Web IDL's `{{TypeError}}` is basically a redirect to ES. But since these algorithm steps are written in the style of ECMAScript (including the `throw` reference), I'd still prefer to use the ECMAScript link if possible.

Reply:
Ohhh, I missed that distinction, indeed. That can be fixed. (Using `<l spec=ecmascript>{{TypeError}}</l>`.)